dist/newrelic/newrelic.yml

```diff
@@ -0,0 +1,302 @@
+# This file configures the New Relic Agent.  New Relic monitors
```

Comment:
The "defaults" don't match what we currently have configured in New Relic, so they'll almost certainly be overridden as soon as we deploy. If we're not using the defaults anyway, why bother have them at all? Including them in git will only serve to confuse future developers.